tests/e2e/notifications.spec.ts

```
@@ -192,30 +190,46 @@ const setupNotificationPage = async (
   };
 };
 
+let notificationSetup: NotificationSetup | null = null;
+
 test.describe('Notification settings', () => {
   test.skip(!e2eConfig, 'E2E credentials are not configured.');
   test.describe.configure({ mode: 'serial' });
 
-  test('updates notification preferences', async () => {
+  test.beforeAll(async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)